### ISSUE TITLE
PUGSETUP_AUTOCKICKER

### ISSUE THREAD

**Issue by Ishaan-Kumar** [splewis/csgo-pug-setup#326]
Right now, we can enable if we can immune admins or not

Can we immune only certains players having flag "b" or flag "o" or any other custom flag

**Comment by splewis** [comment]
Doesn't sm_pugsetup_admin_flag do this?

**Comment by splewis** [comment]
https://github.com/splewis/csgo-pug-setup/blob/master/scripting/pugsetup_autokicker.sp#L51-L53

**Comment by Ishaan-Kumar** [comment]
thing is I want certain players to immune from auto kick but I dont want to give them admin rights

**Comment by splewis** [comment]
You could use a custom admin flag: https://wiki.alliedmods.net/Adding_Admins_(SourceMod)#Levels. Then they won't have admin rights.

**Comment by Ishaan-Kumar** [comment]
// Admin flag to mark players as having elevated permissions - e.g. can always pause,setup,end the game, etc.
// -
// Default: "b"
sm_pugsetup_admin_flag "b"

If I change that to this custom flag let say "t" and the players having flag "t" will not get kicked and doesnt have admin rights now? But it says "Admin flag to mark players as having elevated permissions" ??

Anything I am misunderstanding?

**Comment by splewis** [comment]
Ah, yes, I see. You want immunity without extra command powers. Yeah, you
can't do that right now. A new cvar in the autokicker plugin would be fine
with me.

**Comment by Ishaan-Kumar** [comment]
Yes I want immunity without the players having extra powers. So can we expect this in future? Add to to do list?
